bin/localstack/mk-s3

```diff
@@ -5,9 +5,16 @@
 source bin/lib.sh
 source bin/localstack/lib.sh
 
+BUCKET_NAME="civiform-local-s3"
+
+check_bucket="s3api get-bucket-location
+  --bucket ${BUCKET_NAME}"
```

Comment:
nit: this can be one line and still be under the 80 char line limit
```suggestion
check_bucket="s3api get-bucket-location --bucket ${BUCKET_NAME}"
```